pack/nodejs/package.json

```diff
@@ -1,12 +1,13 @@
 {
     "name": "@pulumi/kubernetes",
-    "version": "v0.13.1-dev-1528839233-g598db27-dirty",
+    "version": "v0.14.1-dev-1530572194-g218d72d-dirty",
```

Comment:
@ellismg Didn't we do something special in other repos to prevent us from committing dirty versions?

Reply:
Drat. I had meant to remove this.

Reply:
Actually, _should_ I remove this? It seems like we do want to depend on v0.14.

Reply:
This is the version of the package itself.  Whatever we check in here is a lie because the next build will overwrite it with that commit’s hash.

I will submit a PR tomorrow to fix this in a similar way to how we handle the other providers.

So either revert it or don’t it doesn’t matter either way.